media-server/media_server.sh

```
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#
+# Author: Fabian Solano <fabian.solano@ridgerun.com>
+# 
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+
+
+# Load settings
+source config.txt
+
+gstd_not_found_msg="Could not connect to localhost: Connection refused"
+check_gstd=$(gstd-client list_pipelines  2>&1)
+
+# Verify if gstd is reachable
+if [[ $check_gstd == $gstd_not_found_msg ]] ; then
```

This is something we need to fix on gstd. This should return an error.